src/assets/scss/components/_signInForm.scss

```diff
@@ -16,57 +16,65 @@ sign-in-form {
 
   .details-wrapper {
     display: flex;
-    flex-wrap: wrap; // Keep the summaries on the first line and wrap the description to the next line
+    flex-wrap: wrap;
```

Comment:
Looks like a lot of the comments in this file got moved away from the styles that they're referring to.